app/win/record.js

```diff
@@ -0,0 +1,37 @@
+// Application state fallback upon unexpected quit
+const {app} = require('electron');
+const Config = require('electron-config');
+const wins = require('./windows');
+
+// local storage
+const rec = new Config();
+const recordInterval = 2000;
```

Comment:
What do you think about making this configurable? I remember that back in the day I always changes Microsoft Word's `auto save interval` to the lowest possible 😂